specs/phase0/p2p-interface.md

```diff
@@ -561,10 +562,9 @@ The request MUST be encoded according to the encoding strategy.
 The requester MUST close the write side of the stream once it finishes writing the request message.
 At this point, the stream will be half-closed.
 
-The requester MUST wait a maximum of `TTFB_TIMEOUT` for the first response byte to arrive (time to first byte—or TTFB—timeout).
-On that happening, the requester allows a further `RESP_TIMEOUT` for each subsequent `response_chunk` received.
+The requester MUST NOT make more than two concurrent requests with the same ID.
 
-If any of these timeouts fire, the requester SHOULD reset the stream and deem the req/resp operation to have failed.
+If a timeout happens on the requesting side, they SHOULD reset the stream.
```

Comment:
What kind of timeout can still happen? I think there is nothing left because you already removed `TTFB_TIMEOUT` and `RESP_TIMEOUT`. Or you left this line just in case we will want to re-enable this in the future?

Reply:
as a consumer, you can implement whatever timeouts you want (in fact, you probably should) - this merely says what you should do to inform the server that you're no longer interested